changelog/funcliteralsbyref.dd

```diff
@@ -0,0 +1,30 @@
+Function literals can now return by reference
+
+Prior to this release, there was no way to declare a function that would return a reference by using an anonymous function.
+$(BR)(It was possible to return a pointer but it's not worth mentioning.)
```

Comment:
it's --> it was

Reply:
"that's" should work too but it seems so emphatic